Docs/packer seo templates batch1 (#13282)
hashicorp/packer: website/content/docs/templates/legacy_json_templates/user-variables.mdx

---
description: |

>   The `variables` block contains parameterizd values for your templates. Learn how configure `variables` to improve security, portability, and shareability of the template.

Reviewer: ```suggestion
The `variables` block contains parameterized values for your templates. Learn how to configure `variables` to improve a template's security, portability, and shareability.
```

page_title: variables reference for JSON templates
---

# `variables` reference for JSON templates

This topic describes how to use user variables in Packer templates written in JSON.

`@include 'from-1.5/legacy-json-warning.mdx'`

## Description

> The `variables` block ets your templates to be further configured with variables

Reviewer: ```suggestion
The `variables` block lets you further configure a template with variables
```
